### ISSUE TITLE
Suggestion for Album art to show in Discord Rich Presence

### ISSUE THREAD

**Issue by DippyFoxDerg** [digimezzo/dopamine#226]
How about an option for Discord Rich Presence to display the Album art of the Song you're currently listening to.

A visual to help explain what I mean:
[Discord Rich Presence if Show album cover is disabled](https://cdn.discordapp.com/attachments/898382590199890031/952619841532080148/unknown.png)
[Discord Rich Presence if Show album cover is enabled](https://cdn.discordapp.com/attachments/898382590199890031/952621242471571556/unknown.png)

**Comment by digimezzo** [comment]
@DippyFoxDerg That's not possible unfortunately. The discord API doesn't allow sending images. The Dopamine icon that you see in Discord is an icon that is already uploaded to Discord in advance for the "Game" Dopamine. It is not sent to discord in real time.

**Comment by MulverineX** [comment]
@digimezzo that is incorrect, the ImageKey properties in discord Activity as of late support image URLs. See: https://github.com/th-ch/youtube-music/pull/524

**Comment by digimezzo** [comment]
@MulverineX Thanks for pointing that out. I wasn't aware of this. It's still not ideal, as the images need to be hosted somewhere. But as you pointed out, it could use an image url coming from lastfm or other album art source.

**Comment by TurtleP** [comment]
foobar2000 has a plugin [which does this](https://s0hv.github.io/foo_discord_rich/docs/album-artwork/#upload-scripts). It's a fork of the original plugin, but those scripts work quite well, having used them. Hope this provides some extra insight :)
